c2corg_ui/templates/preferences.html

```diff
@@ -0,0 +1,46 @@
+<%!
+from c2corg_common.attributes import activities
+%>
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+<%block name="pagetitle"><title ng-bind="mainCtrl.page_title('Filter preferences')">${show_title('Filter preferences')}</title></%block>
+<%block name="metarobots"><meta name="robots" content="noindex,nofollow"></%block>
+
+<div class="preferences">
+  <section app-preferences class="section">
+    <h1 translate>Filter preferences</h1>
+    <p>
+      <label>
+        <input type="checkbox" ng-model="prefCtrl.followed_only">
+        <span translate>In the homepage feed see only statuses from people you follow</span>
+      </label>
+    </p>
+    <p translate>Here you may set activity and region preferences, used for instance in the activity feed.</p>
+    <div ng-hide="prefCtrl.followed_only">
+      <h2 translate>activities</h2>
```

Comment:
I'm always asking myself what's the best way of doing - translating strings as capital or low string words? in both cases we will use sometimes a Capitalized word and sometimes lowercased words. We should have an angular filter that caps/lowers the letters when needed.